lessonbuilder/components/src/java/org/sakaiproject/lessonbuildertool/model/SimplePageToolDaoImpl.java

```diff
@@ -1893,7 +1896,18 @@ public String getLessonSubPageJSON(final String userId, final boolean isInstruct
 			fields[i+1] = pageIds.get(i);
 		}
 
-		final LessonsSubNavBuilder lessonsSubNavBuilder = new LessonsSubNavBuilder(siteId, canSeeAll);
+		ArrayList groups = new ArrayList();
```

Comment:
Why not type this?
List<String> groups = new ArrayList<>();

Actually:
final List<String> groupIds = siteService.getSite(siteId).getGroupsWithMember(userId).stream().map(Group::getId).collect(Collectors.toList());